lmcache/v1/config.py

```diff
@@ -157,6 +161,7 @@ def from_defaults(
         extra_config: Optional[dict] = None,
         save_unfull_chunk: bool = True,
         blocking_timeout_secs: int = 10,
+        mooncake_lookup_client: Optional[str] = None,
```

Comment:
Can you naming this more abstract so that other backend can implement other `xxx_lookup_client` and can reuse this config?